prisma/schema.prisma

```diff
@@ -188,6 +188,11 @@ model BonusHistory {
   createdAt DateTime @default(dbgenerated("timezone('utc'::text, now())")) @db.Timestamp()
 }
 
+model ApiToken {
+  id          String @id @default(uuid())
```

Comment:
I would prefer to use db-generated uids as they  are longer 
`id String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid`

Reply:
fixed
I thought they were the same v4 uuid's